.env

```
@@ -0,0 +1 @@
+EXPO_PUBLIC_API_URL=http://localhost:3000/api
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)